lib/diagnostic.js

```diff
@@ -13,7 +13,8 @@ const add = function(num1, num2){
 add();
 
 // Question 1:  What is the return value of `add();`?
-let answer1 = null; // replace null with your response
+let answer1 = 3; // or its 1 I don't know if num1 technically changes or not
+// replace null with your response
 
```

Comment:
No I did not, but not because I went oh this will work, but honestly because I wasn't sure how to... I am still wicked struggling with testing in terminal, or at least I would assume I would test in terminal?